src/util/segmentation/boundaries/getBoundingBoxAroundPolygon.js

```diff
@@ -0,0 +1,27 @@
+export default function getBoundingBoxAroundPolygon(vertices, image) {
+  let xMin = Infinity;
+  let xMax = 0;
+  let yMin = Infinity;
+  let yMax = 0;
+  const { width, height } = image;
+
+  vertices.forEach(v => {
+    xMin = Math.min(v[0], xMin);
+    xMax = Math.max(v[0], xMax);
+    yMin = Math.min(v[1], yMin);
+    yMax = Math.max(v[1], yMax);
+  });
+
+  xMin = Math.round(xMin);
+  yMin = Math.round(yMin);
+  xMax = Math.round(xMax);
+  yMax = Math.round(yMax);
+
+  xMax = xMax > width ? width : xMax;
```

Comment:
Can you write these as xMax = Math.min(width, xMax) and xMin = Math.max(0, xMin) (just like it's done within the loop)?

Reply:
Done.